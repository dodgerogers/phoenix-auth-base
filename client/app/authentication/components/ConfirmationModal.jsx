import React from 'react'
import PropTypes from 'prop-types';
import AuthenticationModal from './AuthenticationModal';
import ConfirmationFormContainer from '../containers/ConfirmationFormContainer';


const ConfirmationModal = (props) => {
  return (
    <AuthenticationModal title="Confirm your account" {...props}>
      <ConfirmationFormContainer {...props} />
    </AuthenticationModal>
  );
}

export default ConfirmationModal;
