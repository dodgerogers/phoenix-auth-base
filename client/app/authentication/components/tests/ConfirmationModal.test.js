import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import ConfirmationModal from '../ConfirmationModal';


describe('ConfirmationModal', () => {
  const fullProps = {
    close: jest.fn(),
  };

  const pureComponent = (props) => shallow(
    <ConfirmationModal {...props} />
  );

  it('matches snapshot', () => {
    const wrapper = pureComponent(fullProps);

    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
