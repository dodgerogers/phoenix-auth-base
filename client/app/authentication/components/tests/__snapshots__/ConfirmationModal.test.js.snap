// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ConfirmationModal matches snapshot 1`] = `
<AuthenticationModal
  close={[Function]}
  title="Confirm your account"
>
  <Connect(ReduxForm)
    close={[Function]}
  />
</AuthenticationModal>
`;
